src/components/AddCustomSoundModal.tsx

```diff
@@ -0,0 +1,308 @@
+                      selectedTypes.includes(NOTIFICATION_TYPE.PRAYER)
+                        ? "text-primary mr-3"
+                        : "text-typography-secondary mr-3"
+                    }
+                  />
+                  <Text className="text-typography font-medium flex-1">
+                    {t("notification.type.prayer")}
+                  </Text>
+                </Pressable>
+
+                <Pressable
+                  onPress={() => !isProcessing && toggleType(NOTIFICATION_TYPE.IQAMA)}
+                  disabled={isProcessing}
+                  className={`flex-row items-center p-3 rounded-lg ${
+                    selectedTypes.includes(NOTIFICATION_TYPE.IQAMA)
+                      ? "bg-background-info/20"
+                      : "bg-transparent active:bg-surface-hover"
+                  }`}>
+                  <Icon
+                    as={selectedTypes.includes(NOTIFICATION_TYPE.IQAMA) ? CheckSquare : Square}
+                    size="lg"
+                    className={
+                      selectedTypes.includes(NOTIFICATION_TYPE.IQAMA)
+                        ? "text-primary mr-3"
+                        : "text-typography-secondary mr-3"
+                    }
+                  />
+                  <Text className="text-typography font-medium flex-1">
+                    {t("notification.type.iqama")}
+                  </Text>
+                </Pressable>
+
+                <Pressable
+                  onPress={() => !isProcessing && toggleType(NOTIFICATION_TYPE.PRE_ATHAN)}
+                  disabled={isProcessing}
+                  className={`flex-row items-center p-3 rounded-lg ${
+                    selectedTypes.includes(NOTIFICATION_TYPE.PRE_ATHAN)
+                      ? "bg-background-info/20"
+                      : "bg-transparent active:bg-surface-hover"
+                  }`}>
+                  <Icon
+                    as={selectedTypes.includes(NOTIFICATION_TYPE.PRE_ATHAN) ? CheckSquare : Square}
+                    size="lg"
+                    className={
+                      selectedTypes.includes(NOTIFICATION_TYPE.PRE_ATHAN)
+                        ? "text-primary mr-3"
+                        : "text-typography-secondary mr-3"
+                    }
+                  />
+                  <Text className="text-typography font-medium flex-1">
+                    {t("notification.type.preAthan")}
+                  </Text>
+                </Pressable>
+              </VStack>
+            </VStack>
+
+            {/* Error Message */}
+            {error && (
+              <HStack
+                space="sm"
+                className="bg-background-error border border-border-error rounded-xl p-4">
+                <Icon as={X} size="sm" className="text-error" />
+                <Text size="sm" className="text-error flex-1 font-medium">
+                  {error}
+                </Text>
+              </HStack>
+            )}
+          </VStack>
```

Comment:
Small nitpick, in the model there is a small space after the "available in" selection before the separator

<img width="361" height="841" alt="Image" src="https://github.com/user-attachments/assets/4834bb31-3a02-4cbf-b87f-4748f1442f8b" />

It also exist without the error, not sure which part causes this tbh